docs/architecture/decisions/0008-file-upload.md

```diff
@@ -0,0 +1,67 @@
+# 8. File upload
+
+Date: 2025-09-30
+
+## Status
+
+Accepted
+
+## Context
+
+Our registered users will need to upload a Summary Log that is in Excel. These files come from external users, so there is a possibility that they are unsafe. We need to:
+
+- accept the file upload
+- scan for viruses before we store them
+- store file in a reliable location for later validation/processing
+
+The Core Delivery Platform (CDP) provides a managed Uploader service that does exactly this. It takes care of file acceptance, virus scanning and storage in S3. Using it means we don’t have to build our own file handling pipeline.
+
+## Decision
+
+We will integrate with CDP Uploader for our file uploads.
+
+The flow works like this:
+
+- The frontend asks CDP to start an upload, and CDP gives back an `uploadId` and URL.
+- The operator uploads their Excel file and the frontend sends it to CDP.
+- CDP runs an async virus scan. [See documentation](https://github.com/DEFRA/cdp-uploader/blob/main/README.md)
+- CDP stores the file in our S3 bucket.
+- If the file is clean, CDP calls back to the frontend.
+- Errors are handled through the frontend, either through error on upload component or through an error page
+- The frontend calls the backend with the `uploadId` and the URL.
+- The backend retrieves the file from S3 for further processing.
+
+### Flow Diagram
+
+```mermaid
+sequenceDiagram
```

Comment:
The virus scan block does not make full sense to me, yet. 

Maybe we don't need a more detailed explanation to accept an ADR, though.

@nijk what do you think? Are we good with this level of detail?